packages/eds-core-react/src/components/Breadcrumbs/Breadcrumbs.test.tsx

```diff
@@ -24,6 +25,16 @@ describe('Breadcrumbs', () => {
     )
     expect(asFragment()).toMatchSnapshot()
   })
+  it('Should pass a11y test', async () => {
```

Comment:
Could we also have the same test with `href="#"` on each `Breadcrumb`? They are then rendered as `a` elements and would be nice to try cover that in terms of a11y also.